analysis/analysis.py

```diff
@@ -28,6 +28,45 @@ def load_data_csv(csv_path: Path):
     return out_data
 
 
+def word_time_relations(student_data: list) -> dict:
+    """
+    Takes a list of OrderedDicts representing student data and attempts to aggregate
```

Comment:
Couple of super nitpicky things about this docstring (you're only getting this because this is my only PR to review this morning 😁)
- the output of the data loader is now a list of dicts, not ordered dicts
- I'd avoid waffling language like "attempts to" in this kind of comment. It does what it does.